src/modules/better-table/quill-better-table.ts

```diff
@@ -46,6 +46,10 @@ export default class BetterTable extends Module {
   constructor(quill, options) {
     super(quill, options);
 
+    const toolbar = this.quill.getModule("toolbar");
+    // use custom remove text formatting that doesn't break the table
+    toolbar.addHandler("clean", () => this.removeTextFormatting());
```

Comment:
Nejlepší by bylo vložit filter do tý původní funkce. To nevím jestli je nějak rozumě možné bez zásahu do core. 

Takže druhá nejlpší možnost je prostě zmazat v delte atributy všemu kromě tabulky. (obrázky by to měli přežít tam se zmažej jenom fixed width). Tohle by pak mělo vyřešit i to když nám tam někdo vloží třeba list nebo něco jiného.